projects/genesis-lrt/index.js

```diff
@@ -1,15 +1,44 @@
 const ADDRESSES = require('../helper/coreAssets.json')
 
 async function tvl(api) {
-  if (api.timestamp > 1715181240) return {}
```

Comment:
I thought the team wanted genesis to have 0 tvl after the merge

Reply:
initially yes. But after some discussions if we could mantain Genesis [Native restaking] and Inception [Isolated Restaking] it would be way better

Reply:
ah, got it